iml-gui/crate/src/components/command_modal.rs

```diff
@@ -284,3 +548,638 @@ fn close_button() -> Node<Msg> {
     ]
     .map_msg(Msg::Modal)
 }
+
+async fn fetch_the_batch<T, F, U>(ids: Vec<u32>, data_to_msg: F) -> Result<U, U>
+where
+    T: DeserializeOwned + EndpointName + 'static,
+    F: FnOnce(ResponseDataResult<ApiList<T>>) -> U,
+    U: 'static,
+{
+    // e.g. GET /api/something/?id__in=1&id__in=2&id__in=11&limit=0
+    let err_msg = format!("Bad query for {}: {:?}", T::endpoint_name(), ids);
+    let mut ids: Vec<_> = ids.into_iter().map(|x| ("id__in", x)).collect();
+    ids.push(("limit", 0));
+    Request::api_query(T::endpoint_name(), &ids)
+        .expect(&err_msg)
```

Comment:
it will panic, won't it?

Reply:
It seems yes, but it shouldn't be fatal. I'll check.

Reply:
It does panic, but the error happens only when serde cannot build parameter string `"id__in=1&id__in=2..."` which is  impossible normally.